contracts/ProjectValidation.sol

```diff
@@ -0,0 +1,180 @@
+pragma solidity ^0.4.10;
+
+import "zeppelin-solidity/contracts/token/SimpleToken.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+contract ProjectValidation {
+
+    enum Stages {
+        collectingSignatures,
+        checkerWork,
+        readyToComplete,
+        closing,
+        successfullyClosed,
+        unsuccessfullyClosed
+    }
+
+    struct Checker {
```

Comment:
Could you please describe why do you need all these fields in struct?

Reply:
1) signature:
  if (signature = true) => project = completed, 
  if (signature = false) => project = nonCompleted
2) signed: this variable shows the fact of signing by checker
3) presence: shows the presence of checker. presence = true when we call checker into the project
I think that these variables in one way or another are connected with checker and I aggregated them into struct for more convenience